core/shared/src/main/scala/spire/std/float.scala

```diff
@@ -6,7 +6,7 @@ import spire.math.Rational
 
 import java.lang.Math
 
-trait FloatIsField extends Field[Float] {
+trait FloatIsField extends Field.WithDefaultGCD[Float] {
```

Comment:
It should actually be exactly equal; when the denom is 2^n, the operations are exact, as long as they don't overflow the mantissa.